test/http-api/inject/pubsub.js

```diff
@@ -91,17 +91,6 @@ module.exports = (http) => {
     })
 
     describe('/peers', () => {
-      it('returns 200 if no topic is provided', (done) => {
-        api.inject({
-          method: 'GET',
-          url: `/api/v0/pubsub/peers`
-        }, (res) => {
-          expect(res.statusCode).to.equal(200)
-          expect(res.result.Strings).to.be.eql([])
-          done()
-        })
-      })
```

Comment:
This test was no longer valid. I've checked go-ipfs and now it is indeed supposed for IPFS to return the list of peers we are pubsub'ing with when no topic is provided.